src/main/cpp/t09_equals.cpp

```diff
@@ -26,5 +26,19 @@
 using namespace std;
 
 int t09_equals() {
-
+    int n,counter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'counter' is not assigned a value.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4493301)